package.json

```diff
@@ -176,6 +176,7 @@
 			"Fixing the @oclif/core version avoids a large number of peer dep errors since 2.6 adds a prod dep on ts-node and thus a peer dep on @types/node"
 		],
 		"overrides": {
+			"@microsoft/api-extractor>typescript": "$typescript",
```

Comment:
Please add a comment to the section above explaining this overrides purpose and when it can be removed (if known).

Reply:
Good call. Will do.